dll/keyboard/kbdhu/kbdhu.c

```diff
@@ -341,7 +341,7 @@ ROSDATA VSC_LPWSTR extended_key_names[] = {
   { 0x47, L"Home" },
   { 0x48, L"Up" },
   { 0x49, L"Page Up" },
-  { 0x4a, L"Left" },
+  { 0x4b, L"Left" },
```

Comment:
> > `   7  * Thanks to: http://www.barcodeman.com/altek/mule/scandoc.php`
> 
> It looks like that domain does not exist anymore. You may want to mark it as such.

Hmm, well this link is basically in every keyboard layout, I guess it would make sense to fix that in another pr, thanks for pointing it out!